ingestion/tests/integration/sql_server/conftest.py

```diff
@@ -22,24 +23,55 @@
 from ..conftest import ingestion_config as base_ingestion_config
 
 
-@pytest.fixture(scope="module")
```

Comment:
More a question on my end. Wouldn't `scope=module` make it so the fixture would be destroyed at the end of the last test in `sql_server/test_lineage.py` for example?

Reply:
Indeed. With containers a best practice is to have **session** (since its most expensive to setup) level fixtures for the containers and **module** level fixtures for data entities like databases or tables.

Reply:
But for this case the data stays static throughout the tests so its fine like this.